### PR TITLE
Create sv.po

### PR DESCRIPTION
**Pull request by eson57**
Add Swedish translation

### CODE REVIEW

File: CMakeLists.txt [hunk 1]
```diff
@@ -10,6 +10,7 @@ if(CMAKE_GENERATOR MATCHES "Unix Makefiles|Ninja")
 # some LSP servers expect compile_commands.json in the project root
 add_custom_target(
     multivnc-copy-compile-commands ALL
+    install(FILES ${CMAKE_CURRENT_BINARY_DIR}/sv.gmo DESTINATION ${CMAKE_INSTALL_DATADIR}/locale/sv/LC_MESSAGES/ RENAME ${LOCALE_DOMAIN}.mo)
```
Comment (bk138) [inline CMakeLists.txt:13]: This can be removed.

File: po/sv.po [hunk 2]
```diff
@@ -0,0 +1,821 @@
...
+"PO-Revision-Date: 2023-05-24 09:20+0200\n"
+"Last-Translator: Åke Engelbrektson <eson@svenskasprakfiler.se>\n"
+"Language-Team: \n"
+"Language: sv\n"
+"MIME-Version: 1.0\n"
+"Content-Type: text/plain; charset=UTF-8\n"
+"Content-Transfer-Encoding: 8bit\n"
+"Plural-Forms: nplurals=2; plural=(n != 1);\n"
+"X-Generator: Poedit 3.1.1\n"
+
```
Comment (bk138) [inline po/sv.po:20]: Please also add a `translator-credits` entry as in https://github.com/bk138/multivnc/blob/master/po/es.po#L21 so your name automatically shows up in the About dialog.

**Review by bk138 — CHANGES_REQUESTED** [review]
Thanks so much! Just 2 minor comments from my side :-)

File: po/sv.po [hunk 3]
```diff
@@ -0,0 +1,824 @@
...
+"Last-Translator: Åke Engelbrektson <eson@svenskasprakfiler.se>\n"
+"Language-Team: \n"
+"Language: sv\n"
+"MIME-Version: 1.0\n"
+"Content-Type: text/plain; charset=UTF-8\n"
+"Content-Transfer-Encoding: 8bit\n"
+"Plural-Forms: nplurals=2; plural=(n != 1);\n"
+"X-Generator: Poedit 3.1.1\n
+
+"msgid "translator-credits"
```
Comment (bk138) [inline po/sv.po:21]: @eson57 I think you got one quote too much here ;-)

**Review by bk138 — CHANGES_REQUESTED** [review]

File: po/sv.po (hunk 3 above)
Comment (eson57) [reply po/sv.po]: It must be to hot.

**Pull request merged**